New package: RCharan.wiki version 1.0

charanravi-online opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/153706)

charanravi-online committed:
New package: RCharan.wiki version 1.0

charanravi-online commented:
@microsoft-github-policy-service agree

stephengillie commented on manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml:
```diff
@@ -0,0 +1,12 @@
+Publisher: R Charan
+PackageName: wiki
+License: MIT License
+ShortDescription: wiki Setup
```
```suggestion
ShortDescription: A command-line tool to get Wikipedia summaries in your terminal. 
```

stephengillie commented on manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml:
```diff
@@ -0,0 +1,12 @@
+PackageLocale: en-US
+Publisher: R Charan
+PackageName: wiki
+License: MIT License
```
```suggestion
License: Apache-2.0
```
I'm not a lawyer, just copying from the project page. If this shouldn't change, then please feel free to ignore the suggestion.

stephengillie commented:
Verify: 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/f5f9f0b0-ddb0-4485-9d3c-84639e801cc0)

charanravi-online committed:
Update manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml

I agree, this change makes more sense for the project. Hence committing this suggestion.

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

charanravi-online committed:
Update manifests/r/RCharan/wiki/1.0/RCharan.wiki.locale.en-US.yaml

Thanks for pointing this out, the project indeed uses the Apache-2.0 license. Hence committing this change.

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>
